docs/posts/masquerade-as-the-bce-inc-home-hub-4000-on-xgs-pon-with-the-bfw-solutions-was-110.md

```diff
@@ -138,7 +143,7 @@ identifiers are available on the back label of the Home Hub 4000, color-coordina
     | Software Version A         | SGC8210154                   |              | [Version listing]               |
     | Software Version B         | SGC8210154                   |              | [Version listing]               |
     | MIB File                   | /etc/mibs/prx300_1V_bell.ini | :check_mark: | VEIP and more                   |
-    | IP Host MAC Address        | 40:65:A3:FF:A7:B1            |              | :purple_circle: MAC address + 1 |
+    | IP Host MAC Address        | 40:65:A3:FF:A7:B1            |              | :purple_circle: WAN MAC address / LAN MAC address + 1 |
```

Comment:
The attribute already classifies it. The remark should be changed to @MAC since that's what's on the label.